### PR TITLE
kwarg: ffmpeg_loglevel

### PR DESCRIPTION
**Pull request by Wikunia**
## PR Checklist

If you are contributing to `Javis.jl`, please make sure you are able to check off each item on this list:

- [x] Did I update `CHANGELOG.md` with whatever changes/features I added with this PR?
- [x] Did I make sure to only change the part of the file where I introduced a new change/feature?
- [x] Did I cover all corner cases to be close to 100% test coverage (if applicable)?
- [x] Did I properly add Javis dependencies to the `Project.toml` + set an upper bound of the dependency (if applicable)?
- [x] Did I properly add test dependencies to the `test` directory (if applicable)?
- [x] Did I check relevant tutorials that may be affected by changes in this PR?
- [x] Did I clearly articulate why this PR was made the way it was and how it was made?

**How did you address these issues with this PR? What methods did you use?**
I've added a kwarg `ffmpeg_loglevel` which defaults to `"panic"` as before but can be changed to `"info"` or some others.
Additionally I've added it to the issue template such that if ffmpeg fails we get more information.
The changelog and Project.toml are updated to release v0.3.2 when this is merged.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -26,13 +26,14 @@ A clear and concise description of what the bug is.
 
 3. Javis version (i.e output of `] status Javis` in the REPL)
 
-4. Minimum working code example that led to bug:
+5. Minimum working code example that led to bug:
```
Comment (TheCedarPrince) [inline .github/ISSUE_TEMPLATE/bug_report.md:29]: Can you renumber this back to 4?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -26,13 +26,14 @@ A clear and concise description of what the bug is.
...
 
 **Expected Behavior and Actual Behavior**
 
 A clear and concise description of what you expected to happen followed up with an explanation of what actually happened.
 
 **Stacktrace (If Applicable)**
+If the stacktrace includes some ffmpeg error please use `ffmpeg_loglevel = "info"` inside your `render` command.
```
Comment (TheCedarPrince) [inline .github/ISSUE_TEMPLATE/bug_report.md:36]: Maybe an exact example:

```julia
render(your_video, "your_animation.gif", ffmpeg_loglevel = "info")
```

**Review by TheCedarPrince — CHANGES_REQUESTED** [review]
Two small changes but feel free to merge when done because it is fine otherwise.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (Wikunia) [reply .github/ISSUE_TEMPLATE/bug_report.md:29]: Damn yeah
